File: app/src/main/java/com/wayfair/themoviedb/feature/detail/DetailFragment.java
package com.wayfair.themoviedb.feature.detail;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.pubbix.base.BaseFragment;

import org.jetbrains.annotations.NotNull;

import java.util.UUID;

public class DetailFragment extends BaseFragment<DetailPresenter> implements DetailContract.View {


    public static Fragment newInstance() {
        Bundle bundle = new Bundle();
        bundle.putString("instance_id", UUID.randomUUID().toString());
        Fragment fragment = new DetailFragment();
        fragment.setArguments(bundle);
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return super.onCreateView(inflater, container, savedInstanceState);
    }

    @Override
    public void showMessage(@NotNull String message) {

    }
}

File: app/src/main/java/com/wayfair/themoviedb/feature/detail/DetailFragment.kt
package com.wayfair.themoviedb.feature.detail

import android.content.Context
import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.pubbix.base.BaseFragment
import java.util.*

class DetailFragment : BaseFragment<DetailPresenter>(), DetailContract.View {

    override fun onAttach(context: Context?) {
        super.onAttach(context)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return super.onCreateView(inflater, container, savedInstanceState)
    }

    override fun showMessage(message: String) {

    }

    companion object {


        fun newInstance(): Fragment {
            val bundle = Bundle()
            bundle.putString("instance_id", UUID.randomUUID().toString())
            val fragment = DetailFragment()
            fragment.arguments = bundle
            return fragment
        }
    }
}
